cli/wadltest/src/main/scala/com/rackspace/com/papi/components/checker/cli/WadlTest.scala

```diff
@@ -223,6 +224,10 @@ object WadlTest {
              You can pass an '$B$param$R' query paramater to the
              request to have the service echo the body of the request
              in the response.
+
+             You can pass an '$B$param2$R' query paramater to the
```

Comment:
This will print "...an **respType** query..." in stead of "..._a_ **respType** query...".